i18n/locales/fr.json

```diff
@@ -0,0 +1,774 @@
+    "81893F06-7D2D-43FE-B13B-098C335C9A3B": "Atlantique du Nord-Ouest",
+    "41778E79-EC55-4717-B500-165999F07D74": "Atlantique du Sud-Est",
+    "800F52F9-E7A7-47F2-94E4-EA34F60A13A1": "Sud de l'océan Indien",
+    "CA3259CC-2FA6-4E8C-8560-967DB05C5D48": "Pacifique Sud-Ouest",
+    "B58E1174-BA60-481A-83EA-CD0310DCFCFD": "Caraïbes élargies et centre ouest de l'Atlantique",
+    "48C4C187-A6A6-4F36-9024-DE73E2BB0721": "Méditerranée",
+    "D9AE82B7-6033-4EBE-9A7D-88A115AEF6FE": "Nord-est de l'océan Indien",
+    "A2AFD051-BE56-4EB6-880B-204A647C876E": "Nord-Ouest de l'océan Indien et zones adjacentes du Golfe",
+    "C540FD85-87E1-4CAC-A37B-27A252A02012": "Mers d'Asie de l'Est",
+    "550938E7-3A09-436C-9A39-CD54277BDDDC": "Mer Noire et mer Caspienne",
+    "4B547476-0A24-4CED-9A35-B4EE9C53EA31": "Mer Baltique",
+    "21B2B800-3F1C-4751-B054-A7430370ACFF": "Océan Atlantique Nord-Est et zones adjacentes",
+    "B91FB56E-DE54-230A-0438-AF7A5A1C3E53": "Communauté de sources hydrothermales sur le versant des îles du Sud-Ouest",
+    "B91FB56E-DE54-230A-0438-AF7A5A1C3E53-DESC": "De nombreux sites d'écosystèmes chimiosynthétiques abritant à la fois des cheminées hydrothermales et des communautés d'infiltration ont été découverts dans cette zone. Le nombre d'espèces de macrofaune et de mégafaune dans la région est le plus élevé de toutes les régions chimiosynthétiques des grands fonds. La présence d'espèces endémiques est également élevée dans cette région : 68 % des espèces y sont endémiques. Cette zone possède toujours son paysage naturel et son environnement en raison de son inaccessibilité.",
+    "EE8FB298-3D1F-E067-B6E6-0D301D2D9F9A": "Crête sud de Reykjanes",
+    "EE8FB298-3D1F-E067-B6E6-0D301D2D9F9A-DESC": "La crête de Reykjanes fait partie de la principale caractéristique topographique de l'océan Atlantique, la dorsale médio-atlantique. La dorsale médio-atlantique sépare les bassins de Terre-Neuve-et-Labrador du bassin de l'Europe occidentale et la mer d'Irminger du bassin islandais, influençant l'hydrographie et la circulation. La crête de la crête est généralement coupée sur toute sa longueur par une profonde vallée du Rift, bordée par de hautes montagnes du Rift, qui sont bordées de hauts plateaux fracturés. Cette région est en grande partie composée de roches volcaniques, qui constituent la base de la zone et fournissent un substrat dur pour la colonisation des communautés benthiques, y compris des espèces vulnérables et formant des habitats. La région abrite plusieurs espèces de requins et de raies en voie de disparition et menacées. La dorsale elle-même et son environnement hydrographique complexe contribuent à accroître le mélange vertical et la turbulence, ce qui entraîne une augmentation de la productivité des zones au-dessus de la crête. Le contour de 2 500 m de profondeur est utilisé pour définir les limites de la zone, capturant la majeure partie de la crête et la distribution connue des coraux d'eau profonde.",
```

Comment:
where this coming from? look like record content no? why it is part of the build and not in record content?